src/runtime/database.cpp

```diff
@@ -144,9 +144,10 @@ std::string Database::open(bool wait, bool memory, bool tty) {
     "create table if not exists schema("
     "  version integer primary key);"
     "create table if not exists runs("
-    "  run_id  integer primary key autoincrement,"
-    "  time    integer not null,"
-    "  cmdline text    not null);"
+    "  run_id     integer primary key autoincrement,"
+    "  time       integer not null,"
+    "  cmdline    text    not null,"
+    "  executable text    not null);"
```

Comment:
nit: I'd probably call it executable_path or something like that to indicate that its a file path.

Reply:
Removed